website/server/models/subscriptionPlan.js

```diff
@@ -7,7 +7,7 @@ export let schema = new mongoose.Schema({
   subscriptionId: String,
   owner: {type: String, ref: 'User', validate: [validator.isUUID, 'Invalid uuid.']},
   quantity: {type: Number, default: 1},
-  paymentMethod: String, // enum: ['Paypal','Stripe', 'Gift', 'Amazon Payments', '']}
+  paymentMethod: String, // enum: ['Paypal','Stripe', 'Gift', 'Amazon Payments', 'Google', '']}
```

Comment:
Nit: since you're already changing this line, make the spacing between values consistent.